src/widgets/NotificationUI.js

```diff
@@ -73,6 +73,11 @@ define(function (require, exports, module) {
     const NOTIFICATION_TYPE_ARROW = "arrow",
         NOTIFICATION_TYPE_TOAST = "toast";
 
+    /**
+     * CSS class names for notification styles.
+     * @type {object}
+     * @const
```

Comment:
Using `@enum` here of type string will likely generate the docs for each of the components as we need to specify what styles are needed in the docs too.